rotkehlchen/api/rest.py

```diff
@@ -1396,13 +1396,17 @@ def query_list_of_all_assets(
 
     def get_assets_mappings(self, identifiers: list[str]) -> Response:
         try:
-            asset_mappings = GlobalDBHandler().get_assets_mappings(identifiers)
+            asset_mappings, asset_collections = GlobalDBHandler().get_assets_mappings(identifiers)
```

Comment:
To avoid duplication in the information and so the frontend can group them later. So for each asset that forms a group we give their group id and then the groups names/symbols